backend/app/agents/devrel/nodes/handlers/web_search.py

```
@@ -1,12 +1,15 @@
 import logging
-from app.agents.shared.state import AgentState
+from typing import Dict, Any
+from app.agents.state import AgentState
 from langchain_core.messages import HumanMessage
-from ..prompts.search_prompt import EXTRACT_SEARCH_QUERY_PROMPT
+from ...prompts.search_prompt import EXTRACT_SEARCH_QUERY_PROMPT
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)